plans/local/main.fmf

```diff
@@ -0,0 +1,8 @@
+/basic:
+    summary+: ' (localhost)'
+    discover+:
+        filter: tag:local
+    prepare:
+        - name: Install missing dependencies
```

Comment:
This syntax is invalid. I copied that [in my own playground](https://github.com/martinpitt/python-simpleline/pull/1), and [filed a wrong bug](https://pagure.io/fedora-ci/general/issue/170) as I was misled by the failure output.

`prepare:` is a dict, not a list. See the [documentation](https://tmt.readthedocs.io/en/latest/spec/steps.html#prepare)

Reply:
Thanks a lot for your insight. Interesting, I'm pretty sure that worked locally :D

Reply:
At the end this syntax is supported, see:

https://github.com/psss/tmt/blob/master/plans/unit.fmf#L9

Seems that any stage could be specified like this to be able specify multiple different steps. However, it's not really required here.

Unfortunately, I can't find any sign about this in the documentation. Am I missing something @psss?

Reply:
Yes, steps support multiple definitions. This is documented in the [multiple configs](https://tmt.readthedocs.io/en/latest/examples.html#multiple-configs) section and included in the [examples](https://github.com/psss/tmt/blob/master/examples/multiple/basic.fmf) as well. Were would you suggest to add an additional mention about this?

Reply:
@psss
Ohh, thanks a lot for the pointers. I totally missed the ``multiple configs`` section. I guess the problem is that it's just in the ``Examples`` section and not mentioned in the ``Specification part``. So maybe, it would be nice to add this to ``Metadata Specification`` section too with explanation that both variants are valid.